MainWindow code cleanup

liveslice opened the pull request:
- Fix an exception that occurs every time the program is launched since no model is loaded
- Mark various variables as readonly
- Remove some redundant code
- Simplify object initialization

MarsQQ commented:
Looks fine to me, though I'll wait for BabyHamsta to see if the changes are a-okay.

Babyhamsta commented:
Most of it looks good, there are a few reserves with some of the removals of the OnnxModel checks as it could allow users to continue to enable a setting that would then lead to a crash or error due to no model being loaded yet.

I can see some of these fixes being added but not all the changes made in this file. I will attempt to add them by hand with the next update.

liveslice commented:
> Most of it looks good, there are a few reserves with some of the removals of the OnnxModel checks as it could allow users to continue to enable a setting that would then lead to a crash or error due to no model being loaded yet.
> 
> I can see some of these fixes being added but not all the changes made in this file. I will attempt to add them by hand with the next update.

Thanks, this was just every fix I could possibly see to make in that file. Not sure what you mean about the removals of checks though - I just added checks everywhere the OnnxModel is referenced to make sure it was loaded and replaced try-catch with a cleaner if statement.
